build.gradle

```diff
@@ -212,13 +210,23 @@ afterEvaluate {
         maintainer 'OpenDistro for Elasticsearch Team <opendistro@amazon.com>'
         url 'https://opendistro.github.io/elasticsearch/downloads'
         summary '''
-         SQL plugin for OpenDistro for Elasticsearch. 
+         Performance Analyzer plugin for OpenDistro for Elasticsearch. 
          Reference documentation can be found at https://opendistro.github.io/elasticsearch/docs.
     '''.stripIndent().replace('\n', ' ').trim()
     }
 
     buildRpm {
+        arch = 'NOARCH'
+        postInstall file('packaging/rpm/postinst')
         archiveName "${packageName}-${version}.rpm"
         dependsOn 'assemble'
     }
+
+    buildDeb {
+        arch = 'amd64'
```

Comment:
rpm is NOARCH but deb is amd64? Can we keep them consistent?

Reply:
In the package we're using (ospackage), there is no NOARCH option for deb build's arch. See the arch entry in this page: https://github.com/nebula-plugins/gradle-ospackage-plugin/wiki/Deb-Plugin